### PR TITLE
Move from drone to Github Actions for 1.6

### PR DESCRIPTION
**Pull request by mitulshah-suse**
ref issue: https://github.com/rancher/rancher/issues/45377

- setup runners to use custom runners provided by EIO, since using ubuntu-latest is causing diskpressure issues https://github.com/rancherlabs/eio/issues/2338
- removed dapper containers, since it is causing failures in pulling images from docker hub. `resolv.conf` file gets mounted with incorrect entries for dns which cannot be overridden directly. ref: https://github.com/rancher/rke/blob/release/v1.5/dind/dind.go#L77-L78
- `rke remove` kept failing with `directory not empty` errors in docker. Updated the code to first stop the container and then remove it with a pause in between. ref: https://github.com/moby/moby/issues/8203#issuecomment-56625506

### CODE REVIEW

**Review by HarrisonWAffel — DISMISSED** [review]

File: .github/workflows/fossa.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Fossa scan
+
+on:
+    push: 
+      branches:
+        - '*'
```
Comment (jiaqiluo) [inline .github/workflows/fossa.yaml]:
```suggestion
        - 'release/v*'
```

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          name: rke-binaries-${{ github.run_number }}-${{ github.run_attempt }}
+          path: build/bin
+      - name: Create pre-release
+        run: |
+          gh release create ${{ github.ref_name }} -p --verify-tag --title "Pre-release ${{ github.ref_name }}" --notes-file build/bin/rke-k8sversions.txt build/bin/rke*
+
+  github-release:
+    permissions:
+      contents: write
+      id-token: write
```
Comment (jiaqiluo) [inline .github/workflows/workflow.yaml]: Do we need this permission id-token here?  Maybe you forgot to drop it?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow.yaml (hunk 2 above)
Comment (mitulshah-suse) [reply .github/workflows/workflow.yaml]: yep.. forgot to drop it.. updated..

File: .github/workflows/fossa.yaml (hunk 1 above)
Comment (mitulshah-suse) [reply .github/workflows/fossa.yaml]: updated to '**' which should cover all branches

**Review by HarrisonWAffel — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**